machine_learning_hep/data/data_run3/database_ml_parameters_LcJet_pp.yml

```diff
@@ -915,16 +935,30 @@ LcJet_pp:
             # Additional cuts applied before mass histogram is filled
             use_cuts: True
             cuts: [
-                    "mlPromptScore > 0.97",
-                    "mlPromptScore > 0.9",
-                    "mlPromptScore > 0.9",
-                    "mlPromptScore > 0.85",
-                    "mlPromptScore > 0.85",
-                    "mlPromptScore > 0.8",
-                    "mlPromptScore > 0.8",
-                    "mlPromptScore > 0.6",
-                    "mlPromptScore > 0.6",
-                ] # (sel_an_binmin bins) systematics FIXME: Update for new model.
+                   "mlPromptScore > 0.97 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.97 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.9 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.9 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.85 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.85 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.8 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.8 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.6 and fCentFT0M >= 70 and fCentFT0M <= 100",
+                   "mlPromptScore > 0.6 and fCentFT0M >= 70 and fCentFT0M <= 100",
```

Comment:
Why do we want to have these in the standard `LcJet` database?

Reply:
I forgot to remove them, thanks